etl/src/replication/table_sync.rs

```diff
@@ -1,17 +1,19 @@
 use config::shared::PipelineConfig;
-use fail::fail_point;
 use futures::StreamExt;
 use postgres::schema::{TableId, TableName};
 use std::sync::Arc;
 use thiserror::Error;
 use tokio::pin;
 use tokio_postgres::types::PgLsn;
 use tracing::{error, info, warn};
+#[cfg(feature = "failpoints")]
```

Comment:
I added our own `failpoints` feature which activates the `fail/failpoints` feature, in this way we can control better all the surrounding code for failpoints.